project/Build.scala

```diff
@@ -86,7 +86,7 @@ object Build extends sbt.Build {
       )
     )
 
-  lazy val js = metaRx.js
+  lazy val js = metaRx.js.enablePlugins(com.thoughtworks.sbtScalaJsMap.ScalaJsMap)
```

Comment:
Could you move this up and write `.jsConfigure(_.enablePlugins(com.thoughtworks.sbtScalaJsMap.ScalaJsMap))`?